.gitignore

```diff
@@ -1,3 +1,5 @@
 Gemfile.lock
 log/*.log
 doc
+.byebug_history
+tags
```

Comment:
Vim or Emacs? 
BTW, you can configure global `gitignore` for all projects ever to ignore such files